WebApplication/Migrations/20201106124846_migr1.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace WebApplication.Migrations
+{
+    public partial class migr1 : Migration
```

Comment:
If you are naming these classes, please give them some sensible names like Migration1 😄